WalletWasabi.Fluent/Views/AddWallet/Create/RecoveryWordsView.axaml

```diff
@@ -9,7 +9,7 @@
              x:DataType="create:RecoveryWordsViewModel"
              x:Class="WalletWasabi.Fluent.Views.AddWallet.Create.RecoveryWordsView">
   <c:ContentArea Title="{Binding Title}"
-                 Caption="Write down the following recovery words in the correct order and store them in a safe place. They are needed to recover this wallet."
+                 Caption="Write down your recovery words in the correct order and store them in various safe places. They are needed to recover this wallet."
```

Comment:
> in various safe places

This is confusing. This suggest I should be storing it in multiple safe places. How? Should I store half of it in one place and another half another?